buildlib/pr/io_demo/io-demo.yml

```diff
@@ -81,20 +77,15 @@ jobs:
       duration: 120
       interference: 'No'
       tests:
-        "tag match on CX4 with user memh":
-          args: "-z"
-          interface: $(roce_iface_cx4)
-          tls: "rc_x"
-          test_name: tag_umemh_cx4_rc
         "active messages on CX6/DC with user memh":
           args: "-A -z"
           interface: $(roce_iface_cx6)
           tls: "dc_x"
           test_name: am_umemh_cx6_dc
-        "Multi HCAs: tag match on CX4 and CX6 RC ":
-          interface: $(roce_iface_cx4) $(roce_iface_cx6)
+        "Multi HCAs: tag match CX6 RC ":
+          interface: $(roce_iface_cx6)
           tls: "rc_x"
-          test_name: multy_cx6_cx4_rc
+          test_name: multy_cx6_rc
```

Comment:
seems this test is no longer needed, it's not multi-HCA any more
let's add testing over BF3 instead. could be in next PR.

Reply:
Yes, let do it in a separate PR